src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDetails.tsx

```diff
@@ -102,10 +102,14 @@ export default class SentryApplicationDetails extends AsyncView<Props, State> {
     return events.map(e => e.split('.').shift());
   }
 
-  handleSubmitSuccess = data => {
+  handleSubmitSuccess = (data: SentryApp) => {
+    const {app} = this.state;
     const {orgId} = this.props.params;
-    addSuccessMessage(t(`${data.name} successfully saved.`));
-    browserHistory.push(`/settings/${orgId}/developer-settings/`);
+    const verb = app ? 'saved' : 'created';
+    const baseUrl = `/settings/${orgId}/developer-settings/`;
+    const url = app ? baseUrl : `${baseUrl}${data.slug}/`;
+    addSuccessMessage(t(`%s successfully ${verb}.`, data.name));
```

Comment:
By interpolating `verb` the PO file extractor will not be able to produce useful message ID values. Nor will translations be able to translate `verb`.

Reply:
@markstory Is there documentation on how the PO file extractor works? Anyways, I updated the translations to remove interpolation.

Reply:
There isn't much documentation, but we're using the [babel-gettext-extractor](https://github.com/getsentry/babel-gettext-extractor) package we maintain to do it.